static/app/utils/metrics/index.tsx

```diff
@@ -311,6 +317,33 @@ export function stringifyMetricWidget(metricWidget: MetricsQuerySubject): string
   return result;
 }
 
+export function isFormattedMQL(mql: string) {
+  mql.split(new RegExp(/[(){}]/));
+
+  // Regex pattern to match MQL string structure
+  const regex =
+    /^([a-zA-Z]+)\(([^)]+)\)(?:{([^}]+)})?(?:\s+by\s+([^,]+(?:,\s*[^,]+)*))?$/;
+
+  // Executing regex match on the input string
+  const matches = mql.match(regex);
+
+  const [, op, mri, query, groupBy] = matches ?? [];
+
+  if (!op || isMRI(mri)) {
+    return false;
+  }
```

Comment:
won't this return false if I check the result of `getFormattedMQL` as it includes a formatted MRI and not a raw one?

Reply:
nope, because since it is formatted it would not satisfy `isMRI`, but i get that it is confusing, simplified in [9120032](https://github.com/getsentry/sentry/pull/65375/commits/9120032e2074ffff3958dc00992d27d02c67f904) also to reflect taht the mri is a required match group